Bot.py

```diff
@@ -1,47 +1,61 @@
 import logging, os
 from datetime import datetime, timedelta
 from typing import List
-import tweepy, json, pytz, socketio
+import tweepy, json, pytz, socketio, tomli
 from tweepy.errors import Unauthorized as TweepyUnauthorized
 from signal import signal, SIGINT
 
 import RadioIDs
 
-VERSION = "3.0.0"
+VERSION = "3.1.1"
 
 log = logging.getLogger()
 
 
 class Bot:
     """The twitter bot."""
 
-    # Consts
-    CALL_TEXT = "{} second encrypted call at {}"
-    NAMES_TEXT = "#{}: {}"
-    HASHTAGS = "#SeattleEncryptedComms"
-    TWEET_PADDING = 20
-    CONFIG = {
-        "filterCode": [44912, 45040, 45112, 45072, 45136],
-        "filterType": "talkgroup",
-        "filterName": "OpenMHZ",
-        "filterStarred": False,
-        "shortName": "kcers1b",
-    }
-    """ # DEBUG CONFIG TO GET A LOT OF API RESPONSES
-    CONFIG = {
-        "filterCode": "",
-        "filterType": "all",
-        "filterName": "OpenMHZ",
-        "filterStarred": False,
-        "shortName": "kcers1b",
-    } """
-
     def __init__(self) -> None:
         """Initializes the class."""
-        self.callThreshold = int(os.getenv("CALL_THRESHOLD", 1))
-        self.debug = os.getenv("DEBUG", "true").lower() == "true"
-        self.window_minutes = int(os.getenv("WINDOW_M", 5))
-        self.timezone = pytz.timezone(os.getenv("TIMEZONE", "US/Pacific"))
+
+        # Get config
+        try:
+            with open("config.toml", "r") as f:
+                self.config = tomli.load(f)
+
+                # Check for necessary config keys
+                for key in ["radio", "twitter"]:
+                    if key not in self.config:
+                        log.error(f"Missing {key} set up in config")
+                        exit(1)
+
+                # Fill in defaults
+                self.config["general"]["timezone"] = (
+                    self.config["general"]["timezone"]
+                    if "timezone" in self.config["general"]
+                    else "US/Pacific"
+                )
```

Comment:
A lot of these defaults can be simplified in a similar manner:
```suggestion
                self.config["general"]["timezone"] = self.config["general"].get("timezone", "US/Pacific")
```